drivers/include/mrf24j40.h

```diff
@@ -0,0 +1,538 @@
+#define MRF24J40_PSEUDO_STATE_RX_ON        (0x06)
+#define MRF24J40_PSEUDO_STATE_PLL_ON       (0x09)       /**< ready to transmit */
+#define MRF24J40_PSEUDO_STATE_SLEEP        (0x0f)       /**< sleep mode */
+#define MRF24J40_PSEUDO_STATE_BUSY_RX_AACK (0x11)       /**< busy receiving data */
+#define MRF24J40_PSEUDO_STATE_BUSY_TX_ARET (0x12)       /**< busy transmitting data */
+#define MRF24J40_PSEUDO_STATE_RX_AACK_ON   (0x16)       /**< wait for incoming data */
+#define MRF24J40_PSEUDO_STATE_TX_ARET_ON   (0x19)       /**< ready for sending data */
+#define MRF24J40_PSEUDO_STATE_TX_ARET_OFF  (0x19)       /**< ready for sending data */
+#define MRF24J40_PSEUDO_STATE_IN_PROGRESS  (0x1f)       /**< ongoing state conversion */
+/** @} */
+
+/**
+ * @brief   Internal device option flags
+ *
+ * `0x00ff` is reserved for general IEEE 802.15.4 flags
+ * (see @ref netdev2_ieee802154_t)
+ *
+ * @{
+ */
+#define MRF24J40_OPT_SRC_ADDR_LONG  (NETDEV2_IEEE802154_SRC_MODE_LONG)  /**< legacy define */
+#define MRF24J40_OPT_RAWDUMP        (NETDEV2_IEEE802154_RAW)            /**< legacy define */
+#define MRF24J40_OPT_AUTOACK        (NETDEV2_IEEE802154_ACK_REQ)        /**< legacy define */
+
+#define MRF24J40_OPT_CSMA           (0x0100)                            /**< CSMA active */
+#define MRF24J40_OPT_PROMISCUOUS    (0x0200)                            /**< promiscuous mode
+                                                                         *   active */
+#define MRF24J40_OPT_PRELOADING     (0x0400)                            /**< preloading enabled */
+#define MRF24J40_OPT_TELL_TX_START  (0x0800)                            /**< notify MAC layer on TX
+                                                                         *   start */
+#define MRF24J40_OPT_TELL_TX_END    (0x1000)                            /**< notify MAC layer on TX
+                                                                         *   finished */
+#define MRF24J40_OPT_TELL_RX_START  (0x2000)                            /**< notify MAC layer on RX
+                                                                         *   start */
+#define MRF24J40_OPT_TELL_RX_END    (0x4000)                            /**< notify MAC layer on RX
+                                                                         *   finished */
+/** @} */
+
+/**
+ * @brief struct holding all params needed for device initialization
+ */
+typedef struct mrf24j40_params {
+    spi_t spi;              /**< SPI bus the device is connected to */
+    spi_speed_t spi_speed;  /**< SPI speed to use */
+    gpio_t cs_pin;          /**< GPIO pin connected to chip select */
+    gpio_t int_pin;         /**< GPIO pin connected to the interrupt pin */
+    gpio_t sleep_pin;       /**< GPIO pin connected to the sleep pin */
+    gpio_t reset_pin;       /**< GPIO pin connected to the reset pin */
+} mrf24j40_params_t;
+
+/**
+ * @brief   Device descriptor for MRF24J40 radio devices
+ */
+typedef struct {

```

Comment:
I think @phectori is right. @Nenaco I'm wondering how you even compiled this way, or was removal by accident?